pkg/ccl/importccl/import_stmt_test.go

```diff
@@ -4540,3 +4541,135 @@ func TestDisallowsInvalidFormatOptions(t *testing.T) {
 		}
 	}
 }
+
+type importSafeVisitor struct {
+	ctx    *tree.EvalContext
+	isSafe bool
+}
+
+func (v *importSafeVisitor) VisitPre(expr tree.Expr) (bool, tree.Expr) {
+	return v.isSafe, expr
+}
+
+var importSafeFunctions = map[string]struct{}{
+	"current_timestamp": {},
+	"now":               {},
+	"localtimestamp":    {},
+}
+
+func (v *importSafeVisitor) VisitPost(expr tree.Expr) (newNode tree.Expr) {
+	if fn, ok := expr.(*tree.FuncExpr); ok {
+		_, v.isSafe = importSafeFunctions[fn.Func.String()]
```

Comment:
Why do not just do fn.Func.Name?

Reply:
I don't think Func has Name member, does it?

Reply:
`fn.Func` is a `ResolvableFunctionReference`. I think we want to ensure it is `Resolve()`'ed (I'm not sure when that happens/if it already happened) and then it will be backed by a `tree.FunctionDefinition`. _That` has `Name` field.

If we don't resolve first, I think the names could still be qualified, which isn't what we want to be using to look in our map?

Reply:
I'd add a testcase for `pg_catalog.now()` to cover this.

Reply:
you're probably right.  Though, i believe column descriptor DefaultExpr has already been resolved; but we should still be defensive

Reply:
It looks like those names are resolved during type checking, which should be done when creating the default column, but also during the Immutable volatility check. A test case would be useful here though.

Reply:
> fn.Func is a ResolvableFunctionReference. I think we want to ensure it is Resolve()'ed (I'm not sure when that happens/if it already happened) and then it will be backed by a tree.FunctionDefinition. _ThathasName` field.

If you're walking a typed expression, then the function reference will be resolved into a function definition, so that's safe to cast to and access the name field.